public/index.html

```diff
@@ -54,6 +54,17 @@
 			</figure>
 		</section>
 
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<footer>
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+			<ul>
+				<li><a href="#">Home</a></li>
```

Comment:
Should we have a link here knowing it's the same page ?

Reply:
Link removed but item kept in order to display the same footer on every page.